### PR TITLE
LPS-165245 It's possible to update Custom Validation outside of Expression Builder

### PR DESCRIPTION
**Pull request by marcelabc**
Follow-up: https://github.com/liferay-objects/liferay-portal/pull/1259

Related issue: https://issues.liferay.com/browse/LPS-165245

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -1091,6 +1092,8 @@ public ObjectEntry updateObjectEntry(
 				objectEntry.getObjectDefinitionId()),
 			objectEntryId, values);
 
+		objectEntryPersistence.clearCache(SetUtil.fromArray(objectEntryId));
+
```
Comment (carolmariaabb) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:1096]: @marcelabc I'm not sure where we should place this.. here or after the `objectEntry = objectEntryPersistence.update(objectEntry);`, for example. Let's wait and see what Brian says.

**Review by carolmariaabb — APPROVED** [review]
LGTM 🎊

**Comment by marcelabc** [comment]
ci:test:sf

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java (hunk 1 above)
Comment (marcelabc) [reply modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:1158]: Hey @carolmariaabb, 
I tested your suggestion, but it did not work as expected.

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
Send to BChan: https://github.com/brianchandotcom/liferay-portal/pull/125503

**Pull request closed without merging**